packages/errors/src/error-codes.ts

```diff
@@ -73,6 +73,7 @@ export enum ErrorCode {
   DUPLICATED_POLICY = 'duplicated-policy',
   TRANSACTION_SQUEEZED_OUT = 'transaction-squeezed-out',
   CONTRACT_SIZE_EXCEEDS_LIMIT = 'contract-size-exceeds-limit',
+  MAX_INPUTS_EXCEEDED = 'max-inputs-exceeded',
```

Comment:
Could you document this new error code in `apps/docs/src/guide/errors/index.md` please.

Reply:
Of course!

Reply:
You diamond 💎